packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -23,6 +23,28 @@ location / {
     include /opt/mesosphere/etc/adminrouter-ui-security.conf;
 }
 
+location /assets {
+    # The UI asset filenames are hashed by content. Hence, they can be cached
+    # by the client indefinitely. Currently, we use a more conservative value.
+    expires 10h;
+    # Some UI assets are pre-compressed. Serve these compressed if the client
+    # accepts gzip encoding.
+    gzip_static on;
+    # Add a "Vary: Accept-Encoding" header to the response to ensure that
+    # a cached gzip file does not get returned to a client that does not
```

Comment:
That cache would be one between the browser and Admin Router, right? Specifically, it's not the browser's cache, right?

Reply:
Yes. the `Vary` header is an instruction for a cache used by multiple clients, typically a web proxy between the browser and the website (Admin Router).